packages/flutter_tools/test/general.shard/macos/cocoapods_test.dart

```diff
@@ -337,11 +337,14 @@ void main() {
     testWithoutContext('exits if Podfile creates the Flutter engine symlink', () async {
```

Comment:
Can we add an catch-all test that breaks without this fix to test that it indeed doesn't rerun pod install if nothing changed?

Reply:
Added.